README.md

```diff
@@ -955,7 +955,17 @@ All statements above will return false if used with `===`
 
 ## Our contributors are working on this section. [Would you like to join?](https://github.com/i0natan/nodebestpractices/issues/256)
 
-<br/><br/><br/>
+<br/><br/>
+
+## ![✔] 7.1. Don't block the event loop
+
+**TL;DR:** Avoid CPU intensive tasks as they will block the mostly single-threaded Event Loop.
```

Comment:
You stated what NOT to do, let's include your recommendation? for example "Avoid CPU intensive tasks as they will block the mostly single-threaded Event Loop **and offload those to a dedicated thread, process or even a different technology based on the context**"